<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 60, 86);</string>
  </property>
  <widget class="QPushButton" name="saqlashButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Saqlash</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uzgartirishButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);
</string>
   </property>
   <property name="text">
    <string>O'zgartirish</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uchirishButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>O'chirish</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ismEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="documentTitle">
    <string>Form</string>
   </property>
   <property name="placeholderText">
    <string>Ism </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="familiyaEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="placeholderText">
    <string>Familiya</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sharifEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Otasining ismi</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>731</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ism</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Familiya</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Otasining ismi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tug'ilgan yili</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Guruh</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Yunalish</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="guruhEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="yunalishEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Bold ITC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(62, 71, 150);</string>
   </property>
   <property name="text">
    <string>Yo`nalishi :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Bold ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(62, 71, 150);</string>
   </property>
   <property name="text">
    <string>Guruhi :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBosh">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Bold ITC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(62, 71, 150);</string>
   </property>
   <property name="text">
    <string>Talaba qo'shish</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tozalashBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Bekor qilish</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Bold ITC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(62, 71, 150);</string>
   </property>
   <property name="text">
    <string>Talabalar Ro`yhati</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
